src/components/dashboard/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'; 
```

Comment:
another file that has been deleted in master, we can also safely remove this as the components now live on App.js